lib/pages/login_page.dart

```diff
@@ -48,44 +49,56 @@ class LoginPage extends StatelessWidget {
     );
   }
 
-  Widget _buildLoginForm() {
+  Widget _buildLoginForm(BuildContext context) {
+    final forgotButtonWidth = 80.0;
+
     return Column(
       mainAxisAlignment: MainAxisAlignment.center,
       children: [
-        CupertinoTextField(
+        TextField(
           autofocus: true,
-          keyboardType: TextInputType.emailAddress,
-          maxLength: 100,
           // TODO localization https://github.com/luongvo/flutter-survey/issues/25
-          placeholder: "Email",
+          decoration: _formInputDecoration(context, "Email"),
+          keyboardType: TextInputType.emailAddress,
+          style: Theme.of(context).textTheme.bodyText1,
           textInputAction: TextInputAction.next,
         ),
         SizedBox(
           height: 20.0,
         ),
         Stack(
           children: [
-            CupertinoTextField(
-              maxLength: 100,
+            TextField(
+              decoration: _formInputDecoration(context, "Password").copyWith(
+                contentPadding: EdgeInsets.symmetric(
+                  horizontal: Dimens.inputHorizontalPadding,
+                  vertical: Dimens.inputVerticalPadding,
+                ).copyWith(
+                  right: forgotButtonWidth,
+                ),
+              ),
               obscureText: true,
-              placeholder: "Password",
+              obscuringCharacter: "●",
```

Comment:
I think this is unnecessary but it's fine for now

Reply:
I would like to produce the same dot size from the design 💡 
![image](https://user-images.githubusercontent.com/16315358/132204944-2b28c72a-8d6c-4219-96e3-92e4f83c1296.png)

instead of the default one without `obscuringCharacter` custommization
![image](https://user-images.githubusercontent.com/16315358/132204989-8231143b-60bd-4d75-84ab-4cb3ed756746.png)